app/helpers/application_helper.rb

```diff
@@ -20,4 +20,8 @@ def fallback_title
   def decorate(model, decorator_class = nil)
     (decorator_class || [model.class, :Decorator].join.demodulize.constantize).new(model)
   end
+
+  def govuk_formatted_date(date)
```

Comment:
In the view `#l(date)` gives you localisation of dates for free. We just have to add the formats to the `config/locales/en.yml`

```yaml
en:
  [...]
  date:
    formats:
      default: '%-d %B %Y'  # 3 January 2019
      short: '%d/%m/%Y'     # 03/01/2019
```

We can add more formats but those 2 are usually the main ones.